framework/wazuh/core/indexer/events.py

```diff
@@ -1,29 +1,57 @@
+import asyncio
 from dataclasses import asdict
+from typing import List
+from uuid import UUID
 
 from .base import BaseIndex
 from wazuh.core.indexer.models.events import Events
+from wazuh.core.indexer.bulk import MixinBulk
 
+from opensearchpy import AsyncOpenSearch
 
-class EventsIndex(BaseIndex):
+
+HARDCODED_EVENTS_INDEX_NAME = "events"
+
+
+class EventsIndex(BaseIndex, MixinBulk):
     """Set of methods to interact with the stateful events indices."""
+    INDEX = HARDCODED_EVENTS_INDEX_NAME
+
+    def __init__(self, client: AsyncOpenSearch):
+        super().__init__(client)
 
-    async def create(self, events: Events) -> dict:
+    async def create(self, events: Events, batcher_client) -> dict:
         """Post new events to the indexer.
 
         Parameters
         ----------
         events : Events
             List of events.
-        
+
         Returns
         -------
         dict
             Indexer response for each one of the events.
         """
-        # TODO(#24713): Implement server to indexer events batching
+        list_of_uid: List[UUID] = []
```

Comment:
```suggestion
        ids: List[UUID] = []
```

Reply:
Fixed in https://github.com/wazuh/wazuh/commit/47e4508357e0fb53f499847d6a791a0f4c86f15b.